amt/charts/dm-manager/templates/configmap.yaml

```diff
@@ -77,12 +78,31 @@ data:
       exit_istio
     fi
 
+    # Check password policy and set AMT password accordingly
+    PASSWORD_POLICY="{{.Values.params.passwordPolicy | default "dynamic" }}"
```

Comment:
This doesn't solve the problem of hard-coded password. The password is in `ConfigMap` and can be retrieved in clear when `ConfigMap` is dumped. It has to move to Kubernetes secret or put directly in vault and mount later.

Reply:
It is not hardcoding, random password is stored to vault directly